src/components/UserDetail/UserDetail.js

```diff
@@ -109,11 +124,40 @@ class UserDetail extends React.Component {
               value={openAccountsLink}
             />
           </Col>
+          <Col xs={4}>
+            <KeyValue
+              label={<FormattedMessage id="ui-checkout.openRequests" />}
+              value={this.renderOpenRequests()}
+            />
+          </Col>
         </Row>
       </div>
     );
   }
 
+  renderOpenRequests() {
+    const {
+      resources,
+      stripes,
+      user,
+    } = this.props;
+
+    const openRequestsCount = get(resources.openRequests, ['records', '0', 'totalRecords'], 0);
+
+    if (!stripes.hasPerm('ui-users.requests.all,ui-requests.all')) return openRequestsCount;
+
+    const openRequestsPath = `/requests?query=${user.barcode}&filters=requestStatus.Open%20-%20Not%20yet%20filled%2CrequestStatus.Open%20-%20Awaiting%20pickup%2CrequestStatus.Open%20-%20In%20transit&sort=Request%20Date`;
```

Comment:
It seems, this line looks too complex for understanding because of its length and symbols like "%20" and "%2". What do you think about it?

Reply:
I agree with you, I also thought about it and decided to follow the similar approach used in the app so far. However, I think I can make it more readable, so I have changed the implementation a bit.